dashboard/src/App.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="app">
+    <div>
+      <Header />
+    </div>
+    <div class="app-body">
+      <Sidebar />
+      <router-view />
+    </div>
+  </div>
+</template>
+
+<script>
+import Header from '@/components/Header.vue';
+import Sidebar from '@/components/Sidebar.vue';
+
+export default {
+  name: 'app',
+  components: { Header, Sidebar },
```

Comment:
Seems like they prefer multi word component names. (https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential) Maybe set them to another name? `main-header` and `main-sidebar` ? Happy with whatever you decide with!